### PR TITLE
fix pebcak

### PR DESCRIPTION
**Pull request by un1versal**
thats what happened my changes slipped in somehow, in any case this fixes that and + 1 missed fix

### CODE REVIEW

File: texturecache.py [hunk 1]
```diff
@@ -2301,7 +2301,7 @@ def jsonWaitForCleanFinished(self, id, method, params):
         if title:
           self.logger.out("Cleaning library: %-9s %5d [%s]\n" % (iType + "id", libraryId, title))
         else:
-          self.logger.out("Cleaning library: %-9s %5d [%s]\n" % (iType + "id", libraryId, title))
+          self.logger.out("Cleaning library: %-9s %5d\n" % (iType + "id", libraryId))
```
Comment (un1versal) [inline texturecache.py]: @MilhouseVH 

This is what your script was before, somehow my chnages slipped in.

When I said my eyes were coned shaped it was no joke :tongue:

**Comment by un1versal** [comment]
if you want I can bump to 2.1.9

**Comment by MilhouseVH** [comment]
I'll do the change locally here, in fact I'll probably leave it until 8196 lands which will hopefully be soon - it's only a cosmetic error for now. Thanks for the heads up.

**Comment by un1versal** [comment]
Theres more missed stuff in this PR, please do those also

**Comment by MilhouseVH** [comment]
Will do.

**Comment by un1versal** [comment]
> I'll do the change locally here, in fact I'll probably leave it until 8196 lands which will hopefully be soon - it's only a cosmetic error for now. Thanks for the heads up.

yw, sorry about that...

Well v16 currently is a bit broken, and its been slow merging fixes.

**Pull request closed without merging**